app/lib/backend_api_logic/routing_policy.rb

```diff
@@ -4,10 +4,14 @@ module BackendApiLogic
   module RoutingPolicy
     def policy_chain
       chain = super
-      return chain unless service.act_as_product?
+      return chain if using_one_regular_backend?
       Builder.new(service).to_a.concat(chain)
     end
 
+    def using_one_regular_backend?
+      backend_api_configs.where { path.not_in(['', '/']) }.count < 2
```

Comment:
I dunno if this will work in Oracle or not but in Oracle the `''` in path is saved as `null`, and I dunno if this will work if it is going to compare `NULL` with this `''`. For me the safest approach is `backend_api_configs.where { path.not_in([nil, '', '/']) }.count < 2` but not sure if necessary and also i dunno if this has a test somewhere 🤔